script/start-gke-env.sh

```diff
@@ -27,12 +30,15 @@ if [[ $(gcloud container clusters list --filter="name:${CLUSTER}") ]]; then
         done
     else
         echo "GKE cluster already exits. Deleting it"
-        gcloud container clusters delete "${CLUSTER}" --zone "${ZONE}"
+        gcloud container clusters delete "${CLUSTER}" --zone "${ZONE}" --quiet
     fi
 fi
 
 echo "Creating cluster ${CLUSTER} in ${ZONE} (v$BRANCH)"
-gcloud container clusters create --cluster-version="${BRANCH}" --zone "${ZONE}" "${CLUSTER}" --num-nodes 2 --machine-type=n1-standard-2 --preemptible --labels=team=kubeapps
+
+# FIXME Removing the use of --labels=team=kubeapps due to a bug in gcloud cli: https://issuetracker.google.com/issues/255708239
```

Comment:
I'd try to stick to the `#TODO(author): foo` format for the TODOs, this way it's easier to identify all the TODOs and who created them.